benchmarks/benchmarks.rb

```diff
@@ -63,8 +63,7 @@ def benchmark_name
 # Use metaprogramming to ensure that each class is created in exactly the
 # the same way.
 BENCHMARK_GEMS.each do |benchmark_gem|
-  # rubocop:disable Security/Eval
-  eval <<-CLASS, binding, __FILE__, __LINE__ + 1
+  eval <<~HEREDOC, binding, __FILE__, __LINE__ + 1 # rubocop:disable Security/Eval
```

Comment:
I don't love `HEREDOC` but was wondering if we might want something more consistent. Right now we use `CLASS`, `END_OF_METHOD`, and `END_OF_METHODS`. I'm happy to revert this change though and keep them as-is, or switch to something more understandable than `HEREDOC`, like `MULTILINE_STRING`.

Thoughts?

Reply:
I am fine with whatever you choose here @JacobEvelyn. Agree that having a consistent usage might be nice.